app/org/maproulette/models/dal/TaskDAL.scala

```diff
@@ -344,6 +344,12 @@ class TaskDAL @Inject()(override val db: Database,
       val result = extractSuggestedFix(element.parent, element.geometries, element.suggestedFix)
       val geometries = result._1
       var suggestedFix = result._2
+
+      if (element.bundleId != None) {
+        SQL(s"UPDATE tasks SET bundle_id = NULL, is_bundle_primary = NULL " +
```

Comment:
I feel like I am reading this one wrong. Why would you set the bundle_id to NULL if the element task that is supplied has a value for the bundleId?